packages/runtime/src/next/api-handler/index.ts

```diff
@@ -15,6 +15,7 @@ export type { Manifest, Font }
 type MakeswiftApiHandlerConfig = {
   appOrigin?: string
   apiOrigin?: string
+  /** @deprecated, use the `fonts` in the ReactRuntime config */
```

Comment:
There should be no comma after `@deprecated`. Let's change the message to this:

```ts
/** @deprecated Use the `fonts` option in the `ReactRuntime` constructor instead. */
```

Reply:
Done!